src/system/SystemPacketBuffer.h

```diff
@@ -148,6 +148,20 @@ struct pbuf
  */
 class DLL_EXPORT PacketBuffer : private pbuf
 {
+public:
+    static constexpr uint16_t h(const char * s)
```

Comment:
should we name this "hash" or something similar? or maybe even "__hash" to make it clear it is internal only? Some explanation on why we do this would be nice: why do we log using the hash of the file and how does one undo such a hash for parsing logs?

Reply:
Oops, that was accidentally left in from troubleshooting on a memory-constrained device where I didn't want source paths in flash. Will remove.